packages/ag-charts-community/src/chart/dom/theme.css

```diff
@@ -70,6 +74,7 @@
     --ag-charts-popover-focus-color: var(--ag-charts-toolbar-focus-color);
     --ag-charts-popover-hover-color: var(--ag-charts-toolbar-hover-color);
     --ag-charts-popover-active-background-color: var(--ag-charts-toolbar-active-background-color);
+    --ag-charts-popover-font-family: var(--ag-charts-toolbar-font-family);
```

Comment:
I would remove this as a separate variable, and have a single place to control font-family across all charts. 

(I can't see a use case for having a different font-family for toolbars to the rest of charts).

Reply:
Updated